modules/olm/crds.tf

```diff
@@ -0,0 +1,17 @@
+module "yaml" {
+  source = "../../../terraform-null-helpers/modules/yaml"
+  # source =
+
+  files = [
```

Comment:
maybe better to have this files in our side and use them by helm chart/provider?
helm supports multiple kind/resource yaml files processing

Reply:
There is no helm module for this and not easy to implement.